lib/puppet-lint/plugins/check_classes/arrow_on_right_operand_line.rb

```diff
@@ -18,26 +18,20 @@ def check
 
   def fix(problem)
     token = problem[:token]
-    tokens.delete(token)
+    remove_token(token)
 
     # remove any excessive whitespace on the line
     temp_token = token.prev_code_token
     while (temp_token = temp_token.next_token)
```

Comment:
I don't see how `next_token_of` can be used in this specific case, as it deletes whitespaces immediately after the prev_code_token. Maybe if a method that deletes a range of tokens is added ...